File: Love Babber/Function/Array/LinearSearch.cpp
#include<iostream>
using namespace std;

bool isPresent(int arr[], int size,int k)
{
    for(int i=0;i<size;i++)
    {
        if(arr[i]==k)
        {
            return true;
        }
    }

    return false;

}

int main()
{
    int n;
    cout<<"enter the size of array"<<endl;;
    cin>>n;
     int arr[n];
     cout<<"Enter the array elements"<<endl;;
     for(int i=0;i<n;i++)
     {
         cin>>arr[i];
     }
     
     int k;
     cout<<"enter the element you want to search"<<endl;
     cin>>k;
     bool status=isPresent(arr,n,k);
     cout<<status;
    return 0;
}

File: Love Babbar java/code/src/Arrays/LinearSearch.java
package Arrays;

import java.util.Scanner;

public class LinearSearch {
    public static void main(String[] args) {
        int n;
        Scanner in = new Scanner(System.in);
        System.out.println("enter the size of array");
        n=in.nextInt();
        int[] arr = new int[n];
        System.out.println("Enter the array elements");
        for(int i=0;i<n;i++)
        {
            arr[i]=in.nextInt();
        }

        int k;
        System.out.println("enter the element you want to search");
       k=in.nextInt();
        boolean status=isPresent(arr,n,k);
        System.out.println(status);
    }

    private static boolean isPresent(int[] arr, int n, int k) {
        for(int i=0;i<n;i++)
        {
            if(arr[i]==k)
            {
                return true;
            }
        }

        return false;
    }

}
